src/main/java/seedu/address/model/lesson/Date.java

```diff
@@ -118,15 +118,21 @@ public Date getPreviousDate(DayOfWeek dayOfWeek) {
         return prevDate;
     }
 
+    //@@author angkoonhwee
     /**
      * Returns true if the date is between the start and end date, inclusive.
+     *
+     * @param start The inclusive start date.
+     * @param end The inclusive end date.
+     * @return True if the date is between the start and end dates, inclusive.
      */
     public boolean isDateBetween(LocalDate start, LocalDate end) {
         boolean isAfterOrEqualsStart = getLocalDate().isAfter(start) || getLocalDate().isEqual(start);
         boolean isBeforeOrEqualsEnd = getLocalDate().isBefore(end) || getLocalDate().isEqual(end);
         return isAfterOrEqualsStart && isBeforeOrEqualsEnd;
     }
 
+    //@@author eeliana
```

Comment:
Can leave empty? Just @@author will do, some of the methods are not written by me